tests/test_core.py

```diff
@@ -1177,16 +1192,18 @@ def test_setjmp_noleak(self):
 '''
     self.do_run(src, r'''ok.''')
 
-  def test_exceptions(self):
```

Comment:
This test is split into two tests: this and `test_exceptions_off`, to make `test_exceptions` work with `@with_both_exception_handling`.